Add steps for Infinite Tracing data sources for browser, mobile, and lambda (#5228)
newrelic/docs-website: src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/configure-serverless-monitoring-aws-lambda.mdx

After you [enable serverless monitoring for AWS Lambda](/docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/enable-aws-lambda-monitoring), you can add additional configuration to fine-tune your data.

## Infinite Tracing [#infinite-tracing]

Distributed tracing is included by default if you set up your functions using Lambda layers.

> If your Lambda functions have some downstream services that use the advanced type of distributed called Infinite Tracing, you should enable Infinite Tracing for your Lambda functions. This ensures that your root span (the initiating Lambda function) is included in the rest of the spans tracked by Infinite Tracing.

Reviewer: ```suggestion
If your Lambda functions communicate with services that use the advanced type of distributed called Infinite Tracing, you should enable Infinite Tracing for your Lambda functions. This ensures that your Lambda spans are included in the rest of the spans tracked by Infinite Tracing.
```

Infinite Tracing uses a trace observer in the cloud to collect all your trace spans. The trace observer uses [tail-based sampling](https://docs.newrelic.com/docs/distributed-tracing/concepts/how-new-relic-distributed-tracing-works/#tail-based) to sort through your traces. This means it waits until all your spans arrive before assembling them into traces. Then, it looks through all your traces to find the important ones. 

> To set up Infinite Tracing for your Lambda functions, go to our [Infinite Tracing](/docs/distributed-tracing/infinite-tracing/set-trace-observer) documentation for steps to create a trace observer and to select which Lambda functions (data sources) should send trace spans to New Relic.

Reviewer: ibid

Author: @jonathanpdx, OK, I fixed this one, too!
